File: code/1011/1011.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split("\n");

const PI = 3.14159;
const r = parseFloat(lines[0]);

console.log(`VOLUME = ${((4.0 / 3.0) * PI * Math.pow(r, 3)).toFixed(3)}`);

File: code/1011/1011.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        double R;
        const double PI = 3.14159;
        double.TryParse(Console.ReadLine(), out R);
        double volume = (4d/3d) * PI * (Math.Pow(R, 3));
        Console.WriteLine("VOLUME = " + volume.ToString("0.000"));
    }
}